services/hanke-service/src/main/resources/db/changelog/changesets/071-grant-delete-user-permission-to-kayttooikeustasot.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+--changeset Topias Heinonen:071-grant-delete-user-permission-to-kayttooikeustasot
+--comment: Grant the MODIFY_USERS permission to all kayttooikeustasot except KATSELUOIKEUS
```

Comment:
Is this comment correct in this context?

Reply:
Sorry, I forgot to edit the comment after copy-pasting it. Thank you for noticing. It should be correct now.